### PR TITLE
Update gene set importer and documentation

### PR DESCRIPTION
**Pull request by sandertan**
- Remove unnecessary info message
- Fix documentation

### CODE REVIEW

File: docs/File-Formats.md [hunk 1]
```diff
@@ -1175,12 +1175,12 @@ geneset_def_version: msigdb_6.1
 ### GSVA score data file
 The data file will be a simple tab separated format, similar to the expression data file: each sample is a column, each gene set a row, each cell contains the GSVA score for that sample x gene set combination.
 
-The first column is the GENESET_ID. This contains the EXTERNAL_ID or "stable id" (MsigDB calls this "standard name") of the gene set. The other colums are the sample columns: An additional column for each sample in the dataset using the sample id as the column header.
+The first column is the `geneset_id`. This contains the name or "stable id" (MSigDB calls this "standard name") of the gene set. The other columns are sample columns: An additional column for each sample in the dataset using the sample id as the column header.
```
Comment (oplantalech) [inline docs/File-Formats.md]: > this contains the name or "stable id"

I guess you mean: this contains the name "stable id"

File: docs/File-Formats.md [hunk 2]
```diff
@@ -1219,12 +1219,12 @@ geneset_def_version: msigdb_6.1
 ### GSVA p-value data file
 The data file will be a simple tab separated format, similar to the score file: each sample is a column, each gene set a row, each cell contains the p-value for the score found for sample x gene set combination.
 
-The first column is the GENESET_ID. This contains the EXTERNAL_ID or "stable id" (MsigDB calls this "standard name") of the gene set. The other colums are the sample columns: An additional column for each sample in the dataset using the sample id as the column header.
+The first column is the `geneset_id`. This contains the name or "stable id" (MSigDB calls this "standard name") of the gene set. The other columns are sample columns: An additional column for each sample in the dataset using the sample id as the column header.
```
Comment (oplantalech) [inline docs/File-Formats.md]: Same error as previous comment.

File: docs/File-Formats.md (hunk 1 above)
Comment (sandertan) [reply docs/File-Formats.md]: @oplantalech I simplified it.

File: core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGenesetHierarchy.java [hunk 3]
```diff
@@ -75,11 +75,6 @@ public void run() {
 
 			// Check if geneset_hierarchy_node already filled
 			boolean emptyDatabase = !DaoGenesetHierarchyNode.checkGenesetHierarchy();
-			if (emptyDatabase) {
```
Comment (fedde-s) [inline core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGenesetHierarchy.java]: Would moving the `emptyDatabase` boolean down to the line on which it is needed break things? And if so, could you add a comment on how `importData()` (which apparently validates something) does interact with `checkGenesetHierarchy()`?

**Comment by sandertan** [comment]
Thanks for the feedback @fedde-s . I improved the readability, can you review again?

**Comment by sandertan** [comment]
Hi @n1zea144 thanks for looking at this PR. I don't think this is a common use case (yet). 

For the related datatype, the gene sets, I added an option to force remove all existing gene sets and add the new ones using a very long parameter(`boolean confirmEmptyGenesets = options.has("confirm-delete-all-genesets-hierarchy-genesetprofiles");`

If this is every required for gene set hierarchy, we can easily add it. Because this hierarchy data is portal wide, we thought the prompt is useful for the most common use case.

**Comment by n1zea144** [comment]
@sandertan Sounds good.

**Comment by sandertan** [comment]
Rebased on master fixed ci.

**Pull request merged**